src/components/Home.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
```

Comment:
For these dropdowns I think it would be a good idea to center all 5 on the screen along with their titles to make it more user friendly. Additionally, let's maybe add a few more pixels of space between the title and the actual selection box.